mapadroid/ocr/screenPath.py

```diff
@@ -696,3 +709,26 @@ async def _take_and_analyze_screenshot(self, delay_after=0.0, delay_before=0.0,
         else:
             returntype, global_dict, self._width, self._height, diff = result
             return returntype, global_dict, diff
+
+    async def track_ptc_login(self, mode="login"):
```

Comment:
* `mode` is not needed anymore - the code is only checking at start of pogo, because I don't think there's any case where we try a PTC login without (re-)starting the app shortly before. So tracking at login time too would count two login attempts when only one happens.
* this function doesn't really belong into `ocr/screenPath.py`, I just slapped it in there when I came up with this idea because it was convenient at the time...

Reply:
Right. I removed mode.
As for the placement, I am open for ideas.